moveit_core/README.md

```diff
@@ -8,3 +8,16 @@ This repository includes core libraries used by MoveIt:
  - interfaces for controllers and sensors
 
 These libraries do not depend on ROS (except ROS messages) and can be used independently.
+
+## Using clang-tidy.
+
+Install clang-tidy with 
+`sudo apt-get install clang-tidy`
+
+To add it to a new package, add `set(CMAKE_EXPORT_COMPILE_COMMANDS ON)` to `CMakeLists.txt`.
+
+Run it on a specific folder, such as `collision_detection`, with
+```
+run-clang-tidy-3.8.py -fix -p=/home/brycew/Desktop/moveit_ws/build/moveit_core/  collision_detection
+```
+
```

Comment:
Thanks for writing this up! I believe this documentation belongs on [this page](http://moveit.ros.org/documentation/contributing/code/) instead

Reply:
Thanks! I had a feeling there was a better place, but couldn't find where.
This has been moved to https://github.com/ros-planning/moveit.ros.org/pull/146